### PR TITLE
F 470 refactor db queries user resolver

### PR DESCRIPTION
**Pull request by FatemeGhasemi**
Relate #470

### CODE REVIEW

**Comment by FatemeGhasemi** [comment]
@mohammadranjbarz  Can you review it?

File: src/resolvers/userResolver.ts [hunk 1]
```diff
@@ -46,7 +52,8 @@ export class UserResolver {
 
   @Query(returns => User, { nullable: true })
   userByAddress(@Arg('address', type => String) address: string) {
-    return this.userRepository.findOne({ walletAddress: address });
+    // return this.userRepository.findOne({ walletAddress: address });
```
Comment (mohammadranjbarz) [inline src/resolvers/userResolver.ts]: Please remove useless comments

File: src/resolvers/userResolver.ts [hunk 2]
```diff
@@ -60,7 +67,8 @@ export class UserResolver {
     @Ctx() { req: { user } }: MyContext,
   ): Promise<boolean> {
     if (!user) throw new Error(errorMessages.AUTHENTICATION_REQUIRED);
-    const dbUser = await User.findOne({ id: user.userId });
+    // const dbUser = await User.findOne({ id: user.userId });
```
Comment (mohammadranjbarz) [inline src/resolvers/userResolver.ts]: Please remove useless comments

**Review by mohammadranjbarz — CHANGES_REQUESTED** [review]
Thanks @FatemeGhasemi 
Very good, I suggest in other resolvers that call user queries, it's better to change them to use your userRepository instead of calling queries directly, for instance this line
https://github.com/Giveth/impact-graph/blob/29ee74f68a15f55b4e45ef3aaffaf4970bed1f07/src/resolvers/projectResolver.ts#L697

File: src/repositories/userRepository.ts [hunk 3]
```diff
@@ -1,6 +1,8 @@
 import { User, UserRole } from '../entities/user';
 
-export const findAdminUserByEmail = async (email): Promise<User | null> => {
+export const findAdminUserByEmail = async (
+  email,
```
Comment (mohammadranjbarz) [inline src/repositories/userRepository.ts]: specify type of `email` in input parameters

Comment (FatemeGhasemi) [reply src/repositories/userRepository.ts]: done!

**Review by mohammadranjbarz — APPROVED** [review]
@FatemeGhasemi Thanks, it seems good to me

**Pull request merged**